tools/library/scripts/generator.py

```diff
@@ -4632,20 +4632,21 @@ def GenerateSM90_TensorOp_fp8_WGMMA_gemm(manifest, cuda_version):
     else:
       assert False, "math inst is not supported"
 
+    # some schedules disabled to save on library size
     if CudaToolkitVersionSatisfies(cuda_version, 12, 1):
       schedules = [
-        [KernelScheduleType.ScheduleAuto, EpilogueScheduleType.ScheduleAuto],
+        # [KernelScheduleType.ScheduleAuto, EpilogueScheduleType.ScheduleAuto],
```

Comment:
Would recommend not to comment the auto schedules.

Reply:
added back for 12.1.